parser/src/cfg/sexpr.rs

```diff
@@ -1,37 +1,57 @@
+use std::iter;
 use std::ops::Index;
 use std::rc::Rc;
 use std::str::Bytes;
-use std::{cmp, iter};
 
-type ParseError = Spanned<String>;
 type HashMap<K, V> = rustc_hash::FxHashMap<K, V>;
 
-type ParseResult<T> = Result<T, ParseError>;
+use super::{ParseError, Result};
 
-use super::error::{span_start_len, CfgError};
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Default)]
```

Comment:
Could make this `Copy` and reduce the amount of `Clone` calls - it's just 3 `usize`s